segment_ytm.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+)
+
+type ytm struct {
+	props   *properties
+	env     environmentInfo
+	service ytmStatusService
+}
+
+const (
+	// YTMDARemoteControlAPIURL is the YTMDA Remote Control API URL property.
+	YTMDARemoteControlAPIURL Property = "ytmda_remote_control_api_url"
+)
+
+func (y *ytm) string() string {
+	// Hit the Remote Control API to find out what song is playing (or paused).
+	// https://github.com/ytmdesktop/ytmdesktop/wiki/Remote-Control-API
+	status, err := y.service.Get()
```

Comment:
you actually already did this when running enabled, to avoid doing this twice, would setting the value in `enabled()` and re-using it here make sense?

Reply:
I wasn't 100% sure about the "lifecycle" of segments. Is `enabled()` called before `string()` every time a "refresh" occurs (new command entered)?

If that is the case, then yes, it makes sense to cache the result when called in `enabled()` and use that result in `string()`.